scripts/get-envoy.bash

```diff
@@ -42,6 +42,7 @@ if [ -f "$_dir/envoy" ]; then
 fi
 
 mkdir -p "$_dir"
-curl -L --compressed -o "$_dir/envoy" "https://github.com/pomerium/envoy-binaries/releases/download/v${_envoy_version}/envoy-${_target}"
+curl -L --compressed -o "$_dir/envoy" \
```

Comment:
```suggestion
curl -L --compressed -o "$_dir/envoy-${_target}" \
```

Reply:
that won't work because the extraction will fail. I updated the script to use an etag check to see if content was modified.